### PR TITLE
Fix #5014 : Add wiki documentation about Triaging process

### LINKED ISSUE #5014: Add wiki documentation about Triaging process
**Is your feature request related to a problem? Please describe.**
As now we have implemented triaging system for oppia-android also by this PR https://github.com/oppia/oppia-android/pull/5013 but we not have proper documentation about Triaging process in oppia-android wiki.

**Describe the solution you'd like**
Add new wiki "Triaging Process" similar to that of oppia-web [wiki](https://github.com/oppia/oppia/wiki/Triaging-process) with just two teams - CLAM and Dev Workflow Android

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation
Fix #5014 : Add wiki documentation about Triaging process

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@seanlip PTAL, Thanks!
I changed some names that from oppia-web team with oppia-android team such as Sandeep or Vojta with Ben or Adhiambo. Is it okay?

File: wiki/Triaging-process.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+6. Determine if the issue is potentially a good first issue
+    - If the issue seems like something a new contributor to Oppia-Android can take up, make sure to add enough information in the issue description for a new contributor to start work on that issue then ask Ben or Adhiambo to add the Good First Issue label. (Note: typically these are “Work: Low” issues.)
+        - For such good first issues, also assign purple “technology: Kotlin”, “technology: Bazel”, “technology: Xml”, etc. labels as appropriate. This helps new contributors identify issues that they would find approachable.
+
+7. After triaging all issues, make sure the issues in the 'Priority' and 'High Priority' buckets are sorted based on severity/priority (high to low).
+
+
+## PM contacts for each team
+- **CLaM**:
+    - Lead: Ben Henning (@BenHenning)
```
Comment (seanlip) [inline wiki/Triaging-process.md]: Actually I'm the PM (product manager) for this team.

File: wiki/Triaging-process.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - Mentor: Adhiambo Peres (@adhiamboperes)
+- **Dev workflow**:
+    - Ben Henning (@BenHenning)
```
Comment (seanlip) [inline wiki/Triaging-process.md]: Ben is the PM for this team.

**Review by seanlip — APPROVED** [review]
@MohitGupta121 Looks good, just a few small comments. Thanks!

File: wiki/Triaging-process.md (hunk 1 above)
Comment (MohitGupta121) [reply wiki/Triaging-process.md]: Done.

File: wiki/Triaging-process.md (hunk 2 above)
Comment (MohitGupta121) [reply wiki/Triaging-process.md]: Done.

**Review by MohitGupta121 — COMMENTED** [review]
@seanlip PTAL, Thanks.
Updated PM for each team, but now only kept PM contacts in team, Is it required to give mentor's contacts or but we can said to Ben or Adhiambo for CLAM team?

**Review by seanlip — APPROVED** [review]
LGTM. @MohitGupta121 Just PM leads is fine, let's just follow what it says on the tin. Thanks!

**Comment by MohitGupta121** [comment]
Okay Thanks @seanlip

**Pull request merged**